TESTING.md

```diff
@@ -11,55 +11,57 @@ This library provides tools to help write tests for code that uses the following
 
-1. Start the local datastore emulator using `LocalGcdHelper`.  This can be done in two ways:
-  - Run `LocalGcdHelper.java`'s `main` method with arguments `START` and (optionally) `--port=<port number>`.  This will create a temporary folder on your computer and bind `localhost:<port number>` for communication with the local datastore.  The port number is an optional argument.  If no port number is specified, port 8080 will be used.
-  - Call `LocalGcdHelper.start(<project ID>, <port number>)` before running your tests.  Save the `LocalGcdHelper` object returned so that you can stop the emulator later.
+1. Start the local Datastore emulator before running your tests using `LocalDatastoreHelper`'s `start` method. This will create a temporary folder on your computer and bind a port for communication with the local datastore. There are two optional arguments for `start`: project ID and consistency. The consistency setting controls the fraction of Datastore writes that are immediately visible in global queries.
+  ```java
+  // Use defaults for project ID and consistency
+  LocalDatastoreHelper helper = LocalDatastoreHelper.start();
+  // or explicitly provide them
+  helper = LocalDatastoreHelper.start("my-project-id", 0.6);
+  ```
 
-2. In your program, create and use a datastore whose host is set host to `localhost:<port number>`.  For example, 
+2. In your program, create and use a `Datastore` object with the options given by the `LocalDatastoreHelper` instance.  For example:
   ```java
-  DatastoreOptions options = DatastoreOptions.builder()
-      .projectId(PROJECT_ID)
-      .host("http://localhost:8080")
-      .build();
-  Datastore localDatastore = options.service();
+  Datastore localDatastore = helper.options().service();
   ```
+  
 3. Run your tests.
 
-4. Stop the local datastore emulator.
-  - If you ran `LocalGcdHelper.java`'s `main` function to start the emulator, run `LocalGcdHelper.java`'s `main` method with arguments `STOP` and (optionally) `--port=<port number>`.  If the port is not supplied, the program will attempt to close the last port started.
-  - If you ran `LocalGcdHelper.start()` to start the emulator, call the `stop()` method on the `LocalGcdHelper` object returned by `LocalGcdHelper.start()`.
+4. Stop the local datastore emulator by calling the `stop()` method, like so:
+  ```java
+  helper.stop();
+  ```
 
 #### On a remote machine
 
-You can test against a remote datastore emulator as well.  To do this, set the `DatastoreOptions` project endpoint to the hostname of the remote machine, like the example below.
+You can test against a remote Datastore emulator as well.  To do this, set the `DatastoreOptions` project endpoint to the hostname of the remote machine, like the example below.
 
   ```java
-  DatastoreOptions options = DatastoreOptions.builder()
-      .projectId(PROJECT_ID)
+  DatastoreOptions options = LocalDatastoreHelper.options()
```

Comment:
`LocalDatastoreHelper.options()` does not seem to exist. I think here we should use:
```java
DatastoreOptions.builder()
        .projectId("my-project-id")
        .host("http://<hostname of machine>:<port>")
        .authCredentials(AuthCredentials.noAuth())
        .build();
```